src/server/api/routes/users.router.js

```diff
@@ -58,10 +57,9 @@ router.get('/role', checkIfAuthenticated, (req, res, next) => {
  * /users/name:
  *   get:
  *     summary: Get a name for specific user by uid
- *     description: Get a name for specific user by uid
- *
- *
- *       Authentication&#58; <code>super_admin</code> <code>admin</code> <code>user</code>
+ *     description: Get a name for specific user by uid <br /><br />
+ *       Authentication&#58; <code>super_admin</code> <code>admin</code><br /><br />
+ *       Authorization&#58;  <code>user</code>
```

Comment:
Why are the colons HTML encoded? Is that really necessary? Further up you use plain colons.

Reply:
I tried to use normal colons during the project and found out that using regular colons inside the text description, breaks the app. I guess YAML does not like that. 

<img width="822" alt="Screen Shot 2020-05-10 at 08 07 07" src="https://user-images.githubusercontent.com/2197501/81492097-78a3bb00-9295-11ea-815d-7ef3f49db1ea.png">

I just did a quick look on SO and found this solution: use quotes around the text descriptions and escape the colons with :: or \
I will implement that and remove the encodings. Good observation.

https://stackoverflow.com/questions/32991517/how-to-escape-colons-and-other-special-characters-in-a-yaml-string

https://yaml.org/spec/1.2/spec.html#id2776092

Reply:
@zkwsk 
I agree with your suggestions for Authentication and Authorization. It makes sense. 
As @cecastosic mentioned, we are not checking roles on this endpoint. But the fact that we are not checking for roles does not mean that it will work without the firebase token. I just tried and I got a 401 in return - not authorized.
That means all endpoints require authentication. The user, be either super_admin, admin or user, have to log in to generate a firebase token, which at that point can be used to access the endpoint documentation. 
I think we should implement your suggestion.

@cecastosic What do you think?

Reply:
Once again, token is related to authentication (not authorization), and ALL endpoints are secured by authentication middleware (ie firebase token). You can choose how you want to document auth part, I am fine with @zkwsk 's solution. IMO is better to put authorization description on the endpoints where it makes sense (add batch and delete batch) and not copy/paste 
Authorization: super_admin, admin, user
on almost all endpoints. 
🙂